scripts/build_updates_img.sh

```diff
@@ -64,23 +67,50 @@ make -e install
 PYTHON_SITE_PACKAGE_DIRS=`${PYTHON_BIN} -c "import site; print(' '.join(site.getsitepackages()))"`
 
 # List of required Python packages
-PY_PACKAGES="dateutil"
+PY_PACKAGES="dateutil ethtool"
```

Comment:
I still got the following errors in rhsm.log when using the RHSM spoke:
```
2019-02-01 10:14:16,206 [WRN] anaconda:2755:MainThread @hwprobe.py:38 - Unable to import the 'ethtool' module.
...
2019-02-01 11:15:34,220 [ERR] anaconda:2755:RegisterConsumerThread @hwprobe.py:772 - 'NoneType' object has no attribute 'get_interfaces_info'
Traceback (most recent call last):
  File "/usr/lib64/python3.7/site-packages/rhsmlib/facts/hwprobe.py", line 685, in get_network_interfaces
    interfaces_info = ethtool.get_interfaces_info(ethtool.get_devices())
AttributeError: 'NoneType' object has no attribute 'get_interfaces_info'
```
because we need the python3-ethtool rpm. Once I added it in the vagrant/roles/pxe-server/defaults/main.yml the error went away.

Reply:
Thanks. I probably installed it manually on pxe-server.